cmd/ww/main.go

```diff
@@ -6,80 +6,194 @@
 package main
 
 import (
+	"context"
+	"crypto/rand"
+	_ "embed"
+	"fmt"
+	"io"
 	"os"
+	"os/signal"
+	"syscall"
 
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/libp2p/go-libp2p/core/crypto"
 	"github.com/lthibault/log"
 	"github.com/urfave/cli/v2"
+	"go.uber.org/fx"
 
-	"github.com/wetware/ww/internal/cmd/cluster"
-	"github.com/wetware/ww/internal/cmd/debug"
-	"github.com/wetware/ww/internal/cmd/run"
-	"github.com/wetware/ww/internal/cmd/start"
-	ww "github.com/wetware/ww/pkg"
+	casm "github.com/wetware/casm/pkg"
+	"github.com/wetware/ww"
+	"github.com/wetware/ww/pkg/anchor"
```

Comment:
Just FYI:  packages that are new to v0.1.0 were placed in the project root directly, rather than as subdirectories of `pkg/`.  Initially, this was just to help me keep track of changes and avoid conflicts.  As we move forward, however, I am tempted to clean this up and adopt the following conventions:  "ordinary application libraries live in the package root; object capabilities live in `pkg/`".

Thoughts?